### PR TITLE
COMMERCE-10173 - It's possible to save a product configuration with wrong values

### PR DESCRIPTION
**Pull request by victorcdp**
Related Issue: [COMMERCE-10173](https://issues.liferay.com/browse/COMMERCE-10173)

Problem was that there was no validation of existence in the inputs.
Proposed fix is to add a required validator so that the product can't be saved with the specified inputs empty.
Also there was no validation at the backend, so this solution includes that as well.

### CODE REVIEW

**Comment by victorcdp** [comment]
ci:test:sf

**Comment by victorcdp** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-service/service.xml [hunk 1]
```diff
@@ -921,6 +921,7 @@
 		<exception>CommerceAddressType</exception>
 		<exception>CommerceAddressZip</exception>
 		<exception>CommerceGeocoder</exception>
+		<exception>CommerceInvalidOrderQuantity</exception>
```
Comment (brianikim) [inline modules/apps/commerce/commerce-service/service.xml]: Use `CommerceOrderQuantity` as exception name. Resend me this PR with the updated change. Thanks

Comment (victorcdp) [reply modules/apps/commerce/commerce-service/service.xml]: should be good now

**Comment by victorcdp** [comment]
ci:test:sf

**Comment by victorcdp** [comment]
ci:test:sf

**Pull request closed without merging**